packages/nimble-components/src/patterns/link/styles.ts

```diff
@@ -0,0 +1,87 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    DigitalGreenDark,
+    PowerGreen,
+    White
+} from '@ni/nimble-tokens/dist/styledictionary/js/tokens';
+import { bodyFont, bodyFontColor } from '../../theme-provider/design-tokens';
+import { Theme } from '../../theme-provider/types';
+import { hexToRgbaCssColor } from '../../utilities/style/colors';
+import { themeBehavior } from '../../utilities/style/theme';
+
+export const linkStyles = css`
+    ${display('inline-flex')}
+
+    :host {
+        box-sizing: border-box;
+        font: ${bodyFont};
+    }
+
+    .control {
+        display: inline;
+        color: var(--ni-private-link-font-color);
+        cursor: default;
+    }
+
+    .control:any-link {
+        cursor: pointer;
+        text-decoration: none;
+    }
+
+    .control:hover {
+        text-decoration: underline;
+    }
+
+    .control:active {
+        color: var(--ni-private-link-active-font-color);
+        text-decoration: underline;
+    }
+`;
+
+export const linkColors = css`
```

Comment:
I think we are at the point where we need to define tokens for the link fonts. These private properties are no longer encapsulated in a single stylesheet or component but are being set and used across multiple with more to come.

Based on the [naming conventions](https://github.com/ni/nimble/blob/main/packages/nimble-components/CONTRIBUTING.md#token-naming) I'd probably go with:
- linkFont
- linkActiveFont
- linkProminentFont
- linkActiveProminentFont

We have an existing [linkStandardFont](https://github.com/ni/nimble/blob/f48bc519c62f8eeb3ab41ec79a07148a0fbcc943/packages/nimble-components/src/theme-provider/design-tokens.ts#L353) that does not seem to be used by anything in nimble. Maybe it should be deleted as part of the change. Outside of nimble it seems to only be used in the [landing page](https://codesearch.natinst.com/search/text?q=link-standard&fold_case=auto&regex=true&context=true) which should probably be updated to use the new anchor elements anyway

Reply:
Added the tokens and am using them. I removed linkStandardFont as suggested.